Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/azavea/spark:1.6.1
+
+ENV VERSION 0.1.0
+
+COPY geop/target/scala-2.10/usace-programanalysis-geop-assembly-${VERSION}.jar /opt/geoprocessing/
+
+WORKDIR /opt/geoprocessing
+
+EXPOSE 8090
```

Comment:
I think that we need to introduce a `VOLUME` directive here to ensure that Spark's scratch space gets its own volume:

```dockerfile
VOLUME /tmp
```

With a volume for `/tmp`:

```bash
$ docker diff 9899                                                    
```

Without a volume for `/tmp`:

```bash
❯ docker diff 8f9                                                     
C /tmp
A /tmp/blockmgr-004afef6-fd3d-4aa7-bbfd-1288eea028a9
C /tmp/hsperfdata_root
A /tmp/hsperfdata_root/1
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105/httpd-79c9aff5-aadf-45cc-ab4d-9dc5edfabcb1
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105/httpd-79c9aff5-aadf-45cc-ab4d-9dc5edfabcb1/files
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105/httpd-79c9aff5-aadf-45cc-ab4d-9dc5edfabcb1/jars
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105/httpd-79c9aff5-aadf-45cc-ab4d-9dc5edfabcb1/jars/usace-programanalysis-geop-assembly-0.1.0.jar
A /tmp/spark-3da5844d-5755-48cf-8e9b-6987b7fb2105/userFiles-bd745ca7-c2e3-4849-a878-e1d42ad999fc
```